### PR TITLE
CDM-16: New Sponsors component

### PR DESCRIPTION
**Pull request by catarinapm**
[WEB]
image

[MOBILE]
image

ps: added overflow-hidden to prevent horizontal scrolling
### Reviewers
- [x] @danielaprado 
- [ ] @mariasbl

### CODE REVIEW

File: fe/app/components/Sponsors/Sponsors.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import Divider from "@components/Divider/Divider";
+import Image from "next/image";
+import { Sponsor, SponsorsProps } from "@components/Sponsors/Sponsors.models";
+
+const Sponsors = ({ sponsors }: SponsorsProps) => {
+  return (
+    <div className="flex flex-col justify-center items-center w-full md:gap-8 gap-3">
+      <Divider />
+      <div className="flex justify-center gap-3 h-5 md:gap-14 md:h-[49px] items-center">
+        {sponsors.map((e: Sponsor) => (
```
Comment (danielaprado) [inline fe/app/components/Sponsors/Sponsors.tsx:10]: Can you update `e` to `sponsor`?

File: fe/app/components/Sponsors/Sponsors.models.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { ImageProps } from "@components/ImageElement/ImageElement.models";
+
+export type Sponsor = Pick<ImageProps, "src" | "alt">;
+
+export interface SponsorsProps {
+  [x: string]: any;
```
Comment (danielaprado) [inline fe/app/components/Sponsors/Sponsors.models.tsx:6]: Is this ever used? 🤔

**Review by danielaprado — CHANGES_REQUESTED** [review]

File: fe/app/components/Sponsors/Sponsors.models.tsx (hunk 2 above)
Comment (catarinapm) [reply fe/app/components/Sponsors/Sponsors.models.tsx:6]: whaaaa wtfuge is this?

File: fe/app/components/Sponsors/Sponsors.tsx (hunk 1 above)
Comment (catarinapm) [reply fe/app/components/Sponsors/Sponsors.tsx:10]: yes sir 👌

**Review by danielaprado — APPROVED** [review]
LGTM 🚀

**Review by mariasbl — APPROVED** [review]

**Pull request merged**